src/js/actions.js

```diff
@@ -326,17 +327,24 @@ export const setGlobalProperties = (appID, menuLayout) => {
     }
 }
 
+export const updateAppStoreConnectionStatus = (isConnected) => {
+    return {
+        type: Actions.UPDATE_APPSTORE_CONNECTION_STATUS,
+        isConnected: isConnected 
+    }
+}
+
 export const updateAvailableAppStoreApps = (availableApps) => {
     return {
         type: Actions.UPDATE_AVAILABLE_APPSTORE_APPS,
         availableApps: availableApps
     };
 }
 
-export const updateInstalledAppStoreApps = (installedApps) => {
+export const updateInstalledAppStoreApps = (installedApp) => {
```

Comment:
`updateInstalledAppStoreApps` and `UPDATE_INSTALLED_APPSTORE_APPS` should be changed too if we're going to change the parameter of the action.

Reply:
@iCollin I think the name `UPDATE_INSTALLED_APPSTORE_APPS` should still apply since it's updating the **list of installed app store apps** by either adding a new `installedApp` or modifying an existing one.